### PR TITLE
Enable diagnostic telemtry for code intel via opt-out site-admin flag.

### PR DESCRIPTION
**Pull request by efritz**
This will start sending back event counts and latencies for code intel usage statics in pings by default.

This must be approved by @dadlerj and we must ensure that the correct documentation is updated prior to this change being available to customers.

### CODE REVIEW

**Comment by efritz** [comment]
@dadlerj I added docs that we'll be tracking event counts. Does anything else need to be done?

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ All notable changes to Sourcegraph are documented in this file.
 
 - experimentalFeatures.splitSearchModes was removed as a site configuration option. It should be set in global/org/user settings.
 - Sourcegraph now waits for `90s` instead of `5s` for Redis to be available before quitting. This duration is configurable with the new `SRC_REDIS_WAIT_FOR` environment variable.
+- Code intelligence usage statistics will be sent back via pings by default. Aggregated event counts and latencies can be disabled via the site admin flag disableEventCountTelemetry.
```
Comment (dadlerj) [inline CHANGELOG.md]:
```suggestion
- Code intelligence usage statistics will be sent back via pings by default. Aggregated event counts and latencies can be disabled via the site admin flag `disableEventCountTelemetry`.
```

Comment (dadlerj) [reply CHANGELOG.md]: This seems to imply that latencies would also be excluded if this was turned on, but that's not the case from below. Was it intended?

Comment (efritz) [reply CHANGELOG.md]: Oh no sorry. This was written before you ok'd latencies being always on. I'll fix it.

File: schema/site.schema.json [hunk 2]
```diff
@@ -141,6 +141,12 @@
       "default": false,
       "group": "Security"
     },
+    "disableEventCountTelemetry": {
```
[the remark this answers is not in the document]
Comment (efritz) [reply schema/site.schema.json]: Renamed.

File: schema/site.schema.json [hunk 3]
```diff
@@ -141,6 +141,12 @@
       "default": false,
       "group": "Security"
     },
+    "disableNonCriticalTelemetry": {
+      "description": "Disable aggregated event counts from being sent back via pings.",
```
Comment (dadlerj) [inline schema/site.schema.json]:
```suggestion
      "description": "Disable aggregated event counts from being sent to Sourcegraph.com via pings.",
```

**Pull request merged**